spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -679,12 +683,18 @@ definitions:
         - h_ellipse:
             type: EstimatedHorizontalErrorEllipse
             desc: The estimated horizontal error ellipse at the user-configured confidence level.
-        - confidence:
+        - confidence_and_geoid:
             type: u8
             desc: Configured confidence level for the estimated position error
```

Comment:
We'll need to update the `desc` field as well and regen all the files again ... 😢

Reply:
Good catch!

Reply:
Updated. How does it look now?